clang/lib/CodeGen/CGCall.cpp

```diff
@@ -712,8 +713,7 @@ CodeGenTypes::arrangeBlockFunctionDeclaration(const FunctionProtoType *proto,
 const CGFunctionInfo &
 CodeGenTypes::arrangeBuiltinFunctionCall(QualType resultType,
                                          const CallArgList &args) {
-  // FIXME: Kill copy.
```

Comment:
Did we actually fix the FIXMEs? They were removed but I don't functionally different code. Which makes sense considering the NFC label.